File: Iniciante/JavaScript/1040 - Media3.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n')


var valores = lines.shift().split(" ")

var nota1 = Number(valores[0])
var nota2 = Number(valores[1])
var nota3 = Number(valores[2])
var nota4 = Number(valores[3])                                       

var media = (((nota1*2) + (nota2*3) + (nota3*4) + (nota4*1))/10)//calculando notas com os pesos
console.log(`Media: ${media.toFixed(1)}`)

if (media >= 5.0 && media <= 6.9){
    console.log('Aluno em exame.')
    var nota5 = Number(lines.shift())
    console.log(`Nota do exame: ${nota5.toFixed(1)}`)

    var recalcular = ((media+nota5)/2)

    if (recalcular >= 5.0){
        console.log('Aluno aprovado.')
    }else {
        console.log('Aluno reprovado.')
    }
    console.log(`Media final: ${recalcular.toFixed(1)}`)
}
if (media>=7.0){
    console.log('Aluno aprovado.')
}if (media < 5.0){
    console.log('Aluno reprovado.')
}

File: Iniciante/Python/1040 - Media3.py
notas = input().split(" ")
n1 = float(notas[0])
n2 = float(notas[1])
n3 = float(notas[2])
n4 = float(notas[3])

media = ((n1*2)+(n2*3)+(n3*4)+(n4*1))/10
print(f'Media: {media:.1f}')
if media >= 7.0:
    print('Aluno aprovado.')
elif media < 5.0:
    print('Aluno reprovado.')
elif media >= 5.0 and media <= 6.9:
    print('Aluno em exame.')
    notaExame = float(input())
    print(f'Nota do exame: {notaExame:.1f}')
    media = (media + notaExame)/2
    if media >= 5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print(f'Media Final: {media:.1f}')

